back/app/models/idea_image.rb

```diff
@@ -25,14 +25,6 @@ class IdeaImage < ApplicationRecord
   mount_base64_uploader :image, IdeaImageUploader
   belongs_to :idea, inverse_of: :idea_images
 
-  validate :image_presence, unless: :skip_image_presence
+  validates :image, presence: true, unless: :skip_image_presence
```

Comment:
I took the opportunity to also remove the unnecessary custom validation from here, first introduced in #3902 since it's closely related to the introduction of a conditional validation to the `LayoutImage` model in this PR.